tests/unit/modules/test_tomcat.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+
+# Import future libs
+from __future__ import absolute_import, print_function, unicode_literals
+
+# Import 3rd-party libs
+from io import BytesIO, StringIO
+
+# Import salt module
+import salt.modules.tomcat as tomcat
+from salt.ext.six import string_types
+
+# Import Salt Testing libs
+from tests.support.mixins import LoaderModuleMockMixin
+from tests.support.mock import MagicMock, patch
+from tests.support.unit import TestCase
+
+
+class TomcatTestCasse(TestCase, LoaderModuleMockMixin):
+    """
+    Tests cases for salt.modules.tomcat
+    """
+
+    def setup_loader_modules(self):
+        return {tomcat: {}}
+
+    def test_tomcat_wget_no_bytestring(self):
+        responses = {
+            "string": StringIO("Best response ever\r\nAnd you know it!"),
+            "bytes": BytesIO(b"Best response ever\r\nAnd you know it!"),
+        }
+
+        string_mock = MagicMock(return_value=responses["string"])
+        bytes_mock = MagicMock(return_value=responses["bytes"])
+        with patch("salt.modules.tomcat._auth", MagicMock(return_value=True)):
```

Comment:
Here is what is breaking the zcbuildout tests.
The `_auth` function in tomcat module either returns `False` that makes the underlying `_wget` to return an error or returns an `opener` that is being directly installed as a default opener to urllib that breaks urllib for further calls.